docker/deployment/scripts/kubeRsyncAssets.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Documentation:
+# -A  agent forwarding
+# -p 55555 this port is being forwarded to 22 on the container
+# -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" these are a hack to bypass security checks for "localhost".  localhost isn't a real host so I don't care about "host key verification failed" errors when doing localhost port forwarding
+
+ssh -A -p 55555 -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" root@localhost "rsync -avzhP chirt@sysops.languageforge.org:/var/www/languageforge.org/htdocs/assets/lexicon /var/www/html/assets/lexicon"
```

Comment:
```suggestion
ssh -A -p 55555 -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" root@localhost "rsync -avzhP chirt@sysops.languageforge.org:/var/www/languageforge.org/htdocs/assets/lexicon /var/www/html/assets/"
```

Reply:
Looks like it's off by one dir, here's from the container:

```sh
root@app-7c76db448d-cmxlw:/var/www/html/assets/lexicon# ls -l 
total 40
drwxrwxr-x 734 www-data www-data 32768 Jul  8 07:30 lexicon
drwxr-xr-x   2 www-data www-data  4096 Jul  5 08:24 sf_test-rmunn-food-02
```

Reply:
actually, just learned you can simply add the trailing `/` as well.

Reply:
thanks for finding this!  I ended up going with the trailing slash option.